features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/SyslogReceiverCamelNettyImpl.java

```diff
@@ -124,6 +132,17 @@ public void setSyslogConnectionHandlers(SyslogConnectionHandler handler) {
     public void run() {
         // Get a log instance
         Logging.putPrefix(Syslogd.LOG4J_CATEGORY);
+        
+        ConsoleReporter reporter = ConsoleReporter.forRegistry(METRICS)
+                .convertRatesTo(TimeUnit.SECONDS)
+                .convertDurationsTo(TimeUnit.MILLISECONDS)
+                .build();
+            reporter.start(1, TimeUnit.SECONDS);
```

Comment:
I have updated the code by removing the console reported and also by changing to interface as well. I am unable to create new pull request since previous pull request still exists and not cancelled.